x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/watcher/actions/ActionWrapper.java

```diff
@@ -174,7 +175,7 @@ public ActionWrapperResult execute(WatchExecutionContext ctx) {
                 Object object = ObjectPath.eval(path, toMap(ctx));
                 int runs = 0;
                 if (object instanceof Collection) {
-                    Collection collection = Collection.class.cast(object);
+                    Collection<?> collection = Collection.class.cast(object);
```

Comment:
I think in this case, we can just say:
```suggestion
                    Collection<?> collection = (Collection<?>) object;
```